### ISSUE TITLE
GG Max - EXP (Version 1) - BUG - VR and pressing TRIGGER to Open Doors / Chests / Pick things up, Fires your gun.

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#4564]
As the title says, When you try the VR, and you are holding a weapon in your hand.
You then try to open a chest, open a door, Pick something up, Everytime you press the TRIGGER when the display says, You will also fire you gun.

**Comment by LeeBamberTGC** [comment]
@Tum1370 Fixed for next EXP build.

**Comment by Tum1370** [comment]
@LeeBamberTGC Hi, Am not sure what changes you have done, but i just tested the VR and these were my findings.
Trigger doesnt do anything for me now. It wont open doors, or pick up items.
The B button on my right controller opens the SHOP even if am not pointing or near the SHOP.
When the SHOP screen opened up, is was actualu opening up right in my face, and impossible really to use.

I tested this using the Valve Index.

I have not tried a gun yet, I only gave this a quick go but this seems worse than before as i cannot actually do anything now in VR without the controllers working correctly.

VR is very buggy in my opinion.
You certainly need to have a setting file so we can choose our own VR Controller settings as nothing you have set seems to work for others.
